projects/qcms/fuzz.options

```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
```

Comment:
I think you don't need this: 
https://github.com/google/oss-fuzz/blob/master/docs/new_project_guide.md#dictionaries
> If the dict filename is the same as your target binary name (i.e. %fuzz_target%.dict), it will be automatically used.